DiamondLightSource/python-artemis
# 386 add slit gaps and smargon motors to s03

Fixes #386, Fixes #377

### To test:
1. Switch your s03_utils to https://gitlab.diamond.ac.uk/controls/python3/s03_utils/-/commits/386_add_slits_to_s03
2. Update your local s03 (`pods/s03/update.sh`)
3. Run new Artemis system tests and confirm they pass
